src/index.js

```diff
@@ -0,0 +1,216 @@
+import fs from 'fs'
```

Comment:
To be honest I'm kinda amazed how all code is generated. Is there a possibility that the code generation will be explained in a doc so contributors might know how it works and where to fix/add stuff if needed?

Reply:
The code is not generated. The package just returns a [proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy) instance that allows us to handle property access dynamically.